test/unit/uk/gov/hmrc/bindingtariffclassification/controllers/CaseControllerSpec.scala

```diff
@@ -188,11 +188,13 @@ class CaseControllerSpec extends UnitSpec with WithFakeApplication with MockitoS
 
     val queueId = Some("valid_queueId")
     val assigneeId = Some("valid_assigneeId")
-    val sortField = SortField.DAYS_ELAPSED
+    // TODO: use all search parameters
```

Comment:
Given there is no logic based on search params (other than the isEmpty check which we can mock) I don’t think testing *all* of them adds any value